include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,508 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
+{
+    /* Vulkan 1.0 Core  */
+    bool robustBufferAccess = false;
+    bool fullDrawIndexUint32 = false;
+    bool imageCubeArray = false;
+    bool independentBlend = false;
+    bool geometryShader    = false;
+    bool tessellationShader = false;
+    //VkBool32    sampleRateShading;
+    //VkBool32    dualSrcBlend;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    bool drawIndirectFirstInstance = false;
+    bool depthClamp = false;
+    bool depthBiasClamp = false;
+    bool fillModeNonSolid = false;
+    bool depthBounds = false;
+    bool wideLines = false;
+    bool largePoints = false;
+    bool alphaToOne = false;
+    bool multiViewport = false;
+    bool samplerAnisotropy = false;
+    //VkBool32    textureCompressionETC2;
+    //VkBool32    textureCompressionASTC_LDR;
+    //VkBool32    textureCompressionBC;
```

Comment:
`IsGLES ? (EXT_Texture_compression_s3tc  && EXT_Texture_compression_s3tc_srgb && EXT_Texture_compression_bptc):true`

The GLES impl cannot consider `NV_Texture_compression_s3tc` as it doesn't provide SRGB format